plugins/cilium-cni/types/types.go

```diff
@@ -110,6 +110,6 @@ type Args struct{}
 // CNI error codes
 // (error codes 100+ are allowed for plugin use)
 const (
-	CniErrHealthzGet uint = 100
-	CniErrUnhealthy       = iota
+	CniErrHealthzGet uint = 100 + iota
```

Comment:
Nice catch, changes all makes sense.  One thing to consider - I wonder if there is any meaningful implication of upgrade to a new version of Cilium if this enum changes?